### DISCUSSION TITLE
blog/2023/11/05/the-biohacker-lifestyle/

### DISCUSSION THREAD

**Discussion by giscus** [extrange/personal-blog#8] [Announcements]
# blog/2023/11/05/the-biohacker-lifestyle/

None

https://nicholaslyz.com/blog/2023/11/05/the-biohacker-lifestyle/

**Comment by SuhYong** [comment]
i had so much fun reading your blogs.. haha what is that blue blocker glasses? i think i need one as well! my house is super dim and i still having insomnia lol

**Reply by extrange** [reply]
Hey SY, thanks for coming by! I found [this comprehensive review](https://www.reddit.com/r/sleephackers/comments/12ci2fp/i_just_finished_testing_30_pairs_of_blueblocking/) of blue blocker glasses which was really helpful, maybe you can find something there which suits you. I ordered from Amazon and it shipped within a week or so.
